bin/client_docker/alpine/Dockerfile

```diff
@@ -18,9 +18,19 @@ MAINTAINER Valdas Petrulis <petrulis.valdas@gmail.com>
 RUN apk add --update --no-cache bash iptables ca-certificates openvpn \
     && rm -rf /var/cache/apk/*
 
-# Install application
-COPY bin/client_docker/docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
-ENTRYPOINT ["/usr/local/bin/docker-entrypoint.sh"]
+# Directories
+RUN mkdir -p /etc/mysterium-client \
```

Comment:
This one is created by `COPY bin/client_package/config /etc/mysterium-client`

Reply:
Fixed